app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -663,6 +705,9 @@ class SamplesHeatmapView extends React.Component {
     let topTaxIds = new Set();
     let topTaxonDetails = {};
     let filteredData = {};
+    // skippedTaxa includes taxa that passed filters but were explicitly
+    // removed by the user, which will still be counted toward the top taxa count.
```

Comment:
Why should we still count these towards the top?

Reply:
This is a user experience/design choice so I'd ultimately like to check with @happyimadesignr  and Maria, but my logic was:

If I am a user and I have a heatmap that says its displaying 100 taxa and I remove one to clean up what I"m seeing, I expect it to now display 99 taxa. However, if we don't count the removed taxon towards the "top 100 taxa" count, the heatmap logic will just look for the next best taxon and silently add it to the heatmap so that it's still displaying the 100 taxa, which could be confusing to the user.

I could also see the other perspective where the user wants to see the top N taxa but explicitly not including specific taxa. Not sure which is more intuitive, so would love to discuss more!

Reply:
For me the latter makes more sense. Select the top 10 given all the filters (including tax id removed).
Imagine the case that the user is not interested in the top 10 and removes them all... It will display an empty heatmap.

Reply:
That makes sense to me! I'll change it.